src/settings.py

```diff
@@ -74,9 +74,15 @@ class Settings(BaseSettings):
     USE_CREDENTIALS: bool = True
     VALIDATE_CERTS: bool = True
 
+    # Директория с шаблонами электронной почты
+    EMAIL_TEMPLATE_DIRECTORY: Path = BASE_DIR / "templates" / "email"
+
     # Адреса электронной почты администраторов
     EMAIL_ADMIN: EmailStr
 
+    # Время жизни токена
+    TOKEN_EXPIRATION: int = 24
```

Comment:
Обычно подсчет идет в секундах, поэтому это может сбить с толку. Давай как и везде поменяем на секунды
```suggestion
    TOKEN_EXPIRATION: int = 60 * 60 * 24
```